osu.Game/Screens/Play/SkipOverlay.cs

```diff
@@ -143,20 +148,29 @@ protected override void LoadComplete()
         /// </summary>
         public void SkipWhenReady()
         {
-            if (skipQueued) return;
+            if (SkipQueued) return;
 
-            skipQueued = true;
+            SkipQueued = true;
             attemptNextSkip();
 
             void attemptNextSkip() => Scheduler.AddDelayed(() =>
             {
+                if (!stillClickable)
+                {
+                    SkipQueued = false;
+                    return;
+                }
+
                 if (!button.Enabled.Value)
                 {
-                    skipQueued = false;
+                    attemptNextSkip();
                     return;
                 }
```

Comment:
`if (!button.Enabled.Value)` condition will never be true because `button.Enabled.Value == stillClickable` which makes previous condition equivalent to this one. Since first condition will return, second one will never be reached.

Reply:
It starts disabled. I changed the initialisation value. Please re-read the diff.